src/site/content/en/blog/conformance/index.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Conformance for Frameworks
+subhead: Conformance in the JavaScript framework ecosystem
+authors:
+  - shubhie
+  - houssein
+date: 2021-06-15
+hero: image/0SXGYLkliuPQY3aSy3zWvdv7RqG2/i8mOAsozYs0H49p7iDOX.jpg
+alt: Yellow and black warnings wall
+
+description: This article describes Conformance, a methodology used in frameworks within Google, and how we plan on open-sourcing it to the JavaScript framework ecosystem
+tags:
+  - blog
+  - aurora
+---
+
+In our [introductory blog post](/aurora), we covered how we've learned a lot while
+building and using frameworks and tools to develop and maintain large scale web applications such as
+Google Search, Maps, Photos, and so on. By safeguarding developers from writing code that can
+negatively affect user experience, we proved that frameworks can play a key role in shifting
+outcomes for performance and application quality.
+
+Internally at Google, we used the term "**Conformance**" to describe this methodology, and this
+article covers how we plan to open-source this concept to the JavaScript framework ecosystem.
+
+## What is Conformance?
+
+At Google, Conformance was an evolution. Teams relied on a small set of deeply experienced
+maintainers who did extensive code reviews, flagging things that impacted app quality and
+maintainability well beyond correctness issues. To scale this to growing teams of app developers, a
+Conformance system was developed to codify best practices in a way that is automated and
+enforceable. This ensured a consistently high bar for app quality and codebase maintainability
+regardless of the number of code contributors.
+
+Conformance is a system that ensures that developers stay on the well-lit path; it builds confidence
+and ensures predictable outcomes. It makes teams productive, and becomes crucial for _scale_ -- as
+teams grow and more features are developed simultaneously. It empowers developers to focus on
+building product features, freeing them from minutiae and the changing landscape in various areas
+such as performance, accessibility, security, etc.
+
+Conformance is founded on **strong defaults** and providing **actionable rules** that can be
+enforced at **authoring time**. This breaks down into the following 3 principles.
+
+{% Aside %}
+Conformance in this article focuses on coding best practices to achieve predictable loading
+performance & [Core Web Vital](/vitals/) scores, but the principles apply equally to
+other aspects such as security, accessibility, and so forth.
+{% endAside %}
+
+### 1. Strong defaults
```

Comment:
could you update this section based on my update in the doc.

Reply:
[Done](https://github.com/GoogleChrome/web.dev/pull/5623/commits/a19b316d9a64929cfb2eec9b76892c0da7a0763e)